scripts/modules/events/base-integration-handler.js

```
@@ -0,0 +1,441 @@
+	 */
+	_calculateDelay(attempt, config) {
+		let delay;
+
+		switch (config.backoffStrategy) {
+			case 'exponential':
+				delay = config.baseDelay * Math.pow(2, attempt - 1);
+				break;
+			case 'linear':
+				delay = config.baseDelay * attempt;
+				break;
+			case 'fixed':
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)